katago_server/startposes/serializers/startpos.py

```diff
@@ -22,3 +22,10 @@ class Meta:
             "run": {"lookup_field": "name"},
         }
 
+    def validate(self,data):
+        if not data["run"]:
```

Comment:
Is the first check needed ?

Reply:
We don't want the server to crash if a user tries to upload a data that even is missing the "run" field, right?

Reply:
I think by defaut model serializer use model field.
Eg use run field from model

By default it will throw ValidationError.

It will do the validation usibng model before validate.


> ![image](https://user-images.githubusercontent.com/13785185/93825908-7a7b8300-fc66-11ea-979e-8203d4f83cb3.png)

Reply:
ok